app/Http/Controllers/Api/User/LoginController.php

```
@@ -40,4 +54,25 @@ public function logout()
         $accessToken->revoke();
         return $this->successResponse(null, Response::HTTP_NO_CONTENT);
     }
+
+    /**
+     * Register user
+     *
+     * @param App\Http\Requests\User\RegisterRequest $request request
+     *
+     * @return json authentication code with user info
+     */
+    public function register(RegisterRequest $request)
+    {
+        try {
+            $input = $request->except('password');
```

ở dưới em đã ghi đè giá trị cái password rồi thì cần gì except password em

---

cái này e lấy thuộc tính 'password' trong $request ra để bcrypt() nó add vô DB cho dong phía dưới đó a